### ISSUE TITLE
My program won't launch the session into the browser.

### ISSUE THREAD

**Issue by ghost** [CookieCadger/CookieCadger#19]
I'm using the latest Firefox, and using CookieCadger 1.06, yet it won't load into the browser. Can anyone help?

**Comment by sullivanmatt** [comment]
Thanks @N0rvar.  Go ahead and run Cookie Cadger from the command line.  Open up a terminal / command prompt and run Cookie Cadger with **java -jar CookieCadger-1.06.jar**.  Now attempt to load a session into the browser.  Does the command line output show any errors or warnings?  If so, please provide them here.  If not, can you detail exactly what happens when you experience this behavior?

**Comment by ghost** [comment]
I have tried the Command script and attempted to load a session, there are no errors. The bar just constantly says " loading request into browser.." etc.

**Comment by sullivanmatt** [comment]
Strange.  Can you provide some platform information, specifically OS, architecture (32 vs 64 bit), and version of Java.  If you let the bar sit and spin at "Loading request into browser" for, say - 5 minutes - does anything ever happen (even up to and including having it simply go away)?

**Comment by ghost** [comment]
Windows 8 (64 bit).
Java update 21 (64 bit).

It just continuously loops, and never ends. Hmm.

**Comment by aadityabhatia** [comment]
Same issue, and here's what I see on the command line:

```
INFO 12/13 21:21:11 o.b.p.j.h.HttpServer - Version Jetty/5.1.x
INFO 12/13 21:21:11 o.b.p.j.u.Container  - Started HttpContext[/,/]
INFO 12/13 21:21:11 o.b.p.j.h.SocketLis~ - Started SocketListener on 0.0.0.0:7878
INFO 12/13 21:21:11 o.b.p.j.u.Container  - Started org.browsermob.proxy.jetty.jetty.Server@6e38c20b
org.openqa.selenium.firefox.NotConnectedException: Unable to connect to host 127.0.0.1 on port 7055 after 45000 ms. Firefox console output:

...

JavaScript error: chrome://browser/content/urlbarBindings.xml, line 654: aUrl is undefined

        at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.start(NewProfileExtensionConnection.java:106)
        at org.openqa.selenium.firefox.FirefoxDriver.startClient(FirefoxDriver.java:244)
        at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:110)
        at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:190)
        at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:183)
        at org.openqa.selenium.firefox.FirefoxDriver.<init>(FirefoxDriver.java:101)
        at com.cookiecadger.BrowserHandler.loadRequestIntoBrowser(BrowserHandler.java:63)
        at com.cookiecadger.CookieCadgerFrame$29$1.doInBackground(CookieCadgerFrame.java:1550)
        at javax.swing.SwingWorker$1.call(SwingWorker.java:296)
        at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
        at java.util.concurrent.FutureTask.run(FutureTask.java:166)
        at javax.swing.SwingWorker.run(SwingWorker.java:335)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:724)
```

**Comment by aadityabhatia** [comment]
Upstream bug report: [selenium #6517: aUrl is undefined](https://code.google.com/p/selenium/issues/detail?id=6517)

**Comment by aadityabhatia** [comment]
I can confirm that upgrading to Selenium 2.38.0 addresses the issue.

**Comment by sullivanmatt** [comment]
My apologies on the delay with addressing this issue.  I'll attend to it this week.

**Comment by tbook** [comment]
Confirming that Selenium 2.39.0 also fixes the problem

**Comment by sullivanmatt** [comment]
Resolved in Cookie Cadger 1.07
